app/code/Magento/Ui/Test/Mftf/Test/AdminGridSearchSelectAllTest.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminGridSearchSelectAllTest">
```

Comment:
I guess it may make more sense to place these tests under the Customer module, since we are using the customer grid on it, I believe shouldn't have tests on Ui that will depend on another module. We can cover the changes on Magento_Ui with Unit tests in this case.